src/Cpanel/CpanelAPI.php

```diff
@@ -292,6 +292,6 @@ public function isAdvancedZoneEditEnabled() {
          * https://documentation.cpanel.net/display/SDK/Guide+to+the+LiveAPI+System+-+The+cpanelfeature%28%29+Method
          * Returns 1 = enabled, 0 = disabled
```

Comment:
that comment is referring to what the cPanel method itself returns, I updated the PHPDoc on the method to reflect the fact that the method returns a Boolean though.